styleguide/Styleguide.js

```diff
@@ -84,8 +85,9 @@ export default class Styleguide extends Component {
                 <Switch>
                   <Route exact path="/" component={ Introduction } />
                   <Route path="/design/colors" component={ Colors } />
-                  <Route path="/design/typography" component={ Typography } />
+                  <Route path="/design/flexibility" component={ Flexibility } />
```

Comment:
`/responsive-design` is probably better than flexibility imo. Is more descriptive 👍